### PR TITLE
docs: add missing documents for test-chain-utils

### PR DESCRIPTION
**Pull request by chuijiaolianying**

### CODE REVIEW

File: util/test-chain-utils/src/chain.rs [hunk 1]
```diff
@@ -83,7 +83,7 @@ lazy_static! {
     };
 }
 
-/// TODO(doc): @chuijiaolianying
+/// Script for load input data hash from input data.
```
Comment (keroro520) [inline util/test-chain-utils/src/chain.rs:86]: Suggest adding `#[doc(hidden)]` for test-used interfaces.

File: util/test-chain-utils/src/median_time.rs [hunk 2]
```diff
@@ -5,7 +5,10 @@ use ckb_types::{
     prelude::*,
 };
 
-/// TODO(doc): @chuijiaolianying
+/// There's a consensus rule to verify that the block timestamp must be larger than
+/// the median timestamp of the previous 37 blocks.
```
Comment (keroro520) [inline util/test-chain-utils/src/median_time.rs:9]: The number of latest blocks to calculate the "median time" is specified via `BlockMedianTimeContext::median_block_count()`. [MockMedianTime says `11`](https://github.com/nervosnetwork/ckb/pull/2469/files#diff-44a2f5373da905a8c7c3fa4ccfdbe233f2e8db21799c417f2889e2460dd7cc22R17-R19)

**Review by keroro520 — CHANGES_REQUESTED** [review]

**Comment by chuijiaolianying** [comment]
@keroro520  Thanks for your suggestions, please take another look

File: util/test-chain-utils/src/chain.rs [hunk 3]
```diff
@@ -83,7 +83,8 @@ lazy_static! {
...
```
Comment (yangby-cryptape) [inline util/test-chain-utils/src/chain.rs:86]: `to do` or `for doing`?

Comment (chuijiaolianying) [reply util/test-chain-utils/src/chain.rs]: Updated, please take another look, thx

**Review by yangby-cryptape — APPROVED** [review]

**Review by doitian — APPROVED** [review]

**Comment by doitian** [comment]
ping @keroro520

**Review by keroro520 — APPROVED** [review]

**Comment by keroro520** [comment]
bors r=doitian,yangby-cryptape,keroro520

**Pull request merged**